### PR TITLE
MySQL: disable foreign key checks

### PR DESCRIPTION
**Pull request by laysauchoa**
Article migration:
https://help.aiven.io/en/articles/5580571-can-i-disable-foreign-key-checks

I've tested all the steps. However, the extra command in the end did not work for me. I could not reproduce.

Fixes: https://github.com/aiven/devportal/issues/657

### CODE REVIEW

File: docs/products/mysql/howto/disable-foreign-key-checks.rst [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    +--------------------+-------+
+    | Variable_name      | Value |
+    +--------------------+-------+
+    | foreign_key_checks | ON    |
+    +--------------------+-------+
+    1 row in set (0.05 sec)
+
+To disable the foreign key checks for the session, you give an additional parameter when you connect to your Aiven for MySQL using the ``mysqlsh``:
+
+.. code::shell
```
Comment (ftisiot) [inline docs/products/mysql/howto/disable-foreign-key-checks.rst:68]: Not sure why but this code is not shown in the preview

File: docs/products/mysql/howto/disable-foreign-key-checks.rst [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+You can connect to your Aiven for MySQL following this command::
+    
+    mysql --user avnadmin --password=USER_PASSWORD --host USER_HOST --port USER_PORT DB_NAME
+
+After connecting, you can run the following command to check the default configuration for your foreign key checks.
+
+.. code:: shell
+
+    mysql> SHOW VARIABLES LIKE 'foreign_key_checks';
```
Comment (ftisiot) [inline docs/products/mysql/howto/disable-foreign-key-checks.rst:53]: Can we only show the actual code people need to copy/paste?
```suggestion
    SHOW VARIABLES LIKE 'foreign_key_checks';
```

**Review by ftisiot — CHANGES_REQUESTED** [review]
Good one Laysa!
The command works for me!

I left some comments to make this article more "devportal" style, also please check the second `mysql` command is not shown in the preview

File: docs/products/mysql/howto/disable-foreign-key-checks.rst [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+You need those variables to run the commands. You can find this information in the **Overview** tab of your on your Aiven for MySQL service, under **MySQL** tab.
+
+.. list-table::
+  :header-rows: 1
+  :widths: 15 60
+  :align: left
+
+  * - Variable
+    - Description
+  * - ``USER_HOST``
```
[the remark this answers is not in the document]
Comment (laysauchoa) [reply docs/products/mysql/howto/disable-foreign-key-checks.rst:33]: I can replace to HOST if it helps

File: docs/products/mysql/howto/disable-foreign-key-checks.rst [hunk 4]
```diff
@@ -0,0 +1,95 @@
+Disable foreign key checks
+==========================
+
+It can be useful to know how to disable foreign key checks. For example, your migration to Aiven for MySQL may face foreign key violations which can result in an error, similar to::
```
[the remark this answers is not in the document]
Comment (laysauchoa) [reply docs/products/mysql/howto/disable-foreign-key-checks.rst]: Changed slightly, but seems good!

**Comment by laysauchoa** [comment]
@ftisiot thanks for the review! addressed the comments. The last command did not work for me as I said on the PR description. So I was not sure if it was correct.

**Comment by ftisiot** [comment]
@laysauchoa is this ready for review?

**Comment by laysauchoa** [comment]
> @laysauchoa is this ready for review?

yes, I have added the last command in the last review, would be possible to check again? thank you

**Review by ftisiot — APPROVED** [review]
LGTM

**Pull request merged**